src/classes/EventHandlers.js

```diff
@@ -326,7 +326,6 @@ class EventHandlers {
 
 		// TODO fuse this into a single call to improve performance
```

Comment:
I think, after the removal of page_domain that TODO is also obsolete (only one call to tabInfo.getTabInfo left).